egs/wsj/s5/steps/dict/train_g2p.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2014  Johns Hopkins University (Author: Yenda Trmal)
+# Apache 2.0
+
+# Begin configuration section.  
+iters=5
+stage=5
+encoding='utf-8'
+remove_tags=true
+only_words=true
+icu_transform="Any-Lower"
+cmd=run.pl
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+[ -f ./path.sh ] && . ./path.sh; # source the path.
+. parse_options.sh || exit 1;
+
+set -u
+set -e
+
+if [ $# != 2 ]; then
+   echo "Usage: $0 [options] <lexicon-in> <work-dir>"
+   echo "    where <lexicon-in> is the training lexicon (one pronunciation per "
+   echo "    word per line) and <word-dir> is directory where the models will "
+   echo "    be stored"
+   echo "e.g.: train_g2p.sh data/local/lexicon.txt exp/g2p/"
+   echo ""
+   echo "main options (for others, see top of script file)"
+   echo "  --iters <int>                                    # How many iterations. Relates to N-ngram order"
+   echo "  --cmd (utils/run.pl|utils/queue.pl <queue opts>) # how to run jobs."
+   exit 1;
+fi
+
+lexicon=$1
+wdir=$2
+
+
+mkdir -p $wdir/log
+
+[ ! -f $lexicon ] && echo "$0: Training lexicon does not exist." && exit 1
+
+if $only_words ; then
+  cat $lexicon | sed 's/^<.*>.*$//g' | sed 's/^#.*//g' > $wdir/lexicon_onlywords.txt
+  lexicon=$wdir/lexicon_onlywords.txt
+fi
+
+if $remove_tags ; then
+  cat $lexicon |\
+    sed 's/_[%|"]//g' | sed 's/_[0-9]\+//g' > $wdir/lexicon_notags.txt
```

Comment:
This, also, is too specific to the BABEL recipe.  I think it would be safer to just leave it out.